src/components/PostList/PostList.jsx

```diff
@@ -1 +1,12 @@
-export const PostList = () => <>Put the list here</>;
+import { PostInfo } from '../PostInfo';
+
+export const PostList = ({ posts }) => (
+  <>
+    <h1 className="App__title">Static list of posts</h1>
```

Comment:
I think this should be part App.jsx

Reply:
> I think this should be part App.jsx

Good point, `App__title` isn't related to the `PostList ` at all